docs/excel/excel-add-ins-pivottables.md

```diff
@@ -101,54 +102,142 @@ Adding the **Farm** column pivots all the sales around each farm. Adding the **T
+
+The following code sample shows a new slicer added to the **Pivot** worksheet. The slicer's source is the **Farm Sales** PivotTable and filters using the **Type** data. The slicer is also named **Fruit Slicer** for future reference.
+
+```js
+Excel.run(async (context) => {
+    var sheet = context.workbook.worksheets.getItem("Pivot");
+    var slicer = sheet.slicers.add(
+        "Farm Sales" /* The slicer data source. For PivotTables, this can be the PivotTable object reference or name. */,
+        "Type" /* The field in the data to filter by. For PivotTables, this can be a PivotField object reference or ID. */
+    );
+    slicer.name = "Fruit Slicer";
+    return context.sync();
+});
+```
+
+### Filter items with a slicer
+
+The slicer filters the PivotTable with items from the `sourceField`. The `Slicer.selectItems` method sets the items that remain in the slicer. These items are passed to the method as a `string[]`, representing the items' keys. Any rows containing those items remain in the PivotTable's aggregation. Subsequent calls to `selectItems` set the list to those calls' keys.
+
+> [!NOTE]
+> If `Slicer.selectItems` is passed an item that's not in the data source, an `InvalidArgument` error is thrown. The contents can be verified through the `Slicer.slicerItems` property, which is a [SlicerItemCollection](/javascript/api/excel/excel.sliceritemcollection).
+
+The following code sample shows three items being selected for the slicer: **Lemon**, **Lime**, and **Orange**.
+
+```js
+Excel.run(async (context) => {
+    var slicer = context.workbook.slicers.getItem("Fruit Slicer");
+    // Anything other than the following three values will be filtered out of the PivotTable for display and aggregation.
+    slicer.selectItems(["Lemon", "Lime", "Orange"]);
+    return context.sync();
+});
+```
+
+To remove filters from the slicer, use the `Slicer.clearFilters` method, as shown in the following sample. Note that this removes all the filters.
+
+```js
+Excel.run(async (context) => {
+    var slicer = context.workbook.slicers.getItem("Fruit Slicer");
+    slicer.clearFilters();
+    return context.sync();
+});
+```
+
+### Style and format a slicer
+
+A slicer's display settings are set through related `Slicer` properties. The following code sample sets the style to **SlicerStyleLight6**, sets the text at the top of the slicer to **Fruit Types**, places the slicer at the position **(395, 15)** on the drawing layer, and the slicer's size to **135x150** pixels.
+
+```js
+Excel.run(async (context) => {
+    var slicer = context.workbook.slicers.getItem("Fruit Slicer");
+    slicer.caption = "Fruit Types";
+    slicer.left = 395;
+    slicer.top = 15;
+    slicer.height = 135;
+    slicer.width = 150;
+    slicer.style = "SlicerStyleLight6";
+    return context.sync();
+});
+```
+
+### Delete a slicer
+
+Slicers are deleted with the `Slicer.delete` method. The following code sample deletes the first slicer from the current worksheet.
```

Comment:
In general, it's best practice to avoid using passive voice like "Slicers are deleted with the `Slicer.delete` method." and instead use active voice like "To delete a slicer, call the `Slicer.delete` method."   Not a huge deal, so I'm not going to comment everywhere that this occurs, but something for you to keep in mind.